Tests/kaas/kaas-sonobuoy-tests/scs-conformance-sonobuoy-plugin.yaml

```diff
@@ -7,7 +7,7 @@ spec:
   command:
   - bash
   args: ["-c","go tool test2json ./custom.test -test.v | tee ${SONOBUOY_RESULTS_DIR}/out.json ; echo ${SONOBUOY_RESULTS_DIR}/out.json > ${SONOBUOY_RESULTS_DIR}/done"]
-  image: ghcr.io/sovereigncloudstack/standards/scs-kaas-conformance:dev
+  image: registry.dnation.cloud/test-mg/scsconformance:dev #ghcr.io/sovereigncloudstack/standards/scs-kaas-conformance:dev
```

Comment:
It seems to me that this is a development artifact. Could you revert these changes?

Reply:
That is true, I will change it.